README.md

```diff
@@ -0,0 +1,82 @@
+# Configurable LTI Consumer XBlock
+
+This XBlock is used to replace edX's lti_consumer xblock. It makes available to
+conceptors in CMS several preconfigured versions of LTI Consumer original
+XBlock. Preconfigured fields are no longer modifiable.
+
+## Installation
+
+This package can be installed with `pip`:
+
+```bash
+$ pip install [--process-dependency-links] configurable_lti_consumer-xblock
+```
+
+Note that the `--process-dependency-links` `pip` option is only required to
+install or test this Xblock as a standalone package. If you plan to install it
+in a base Open edX installation, then you can safely miss this option as the
+only package dependency should already have been resolved.
+
+## Configuration example
+
+The below example configuration instanciate 2 xblocks.
+
+```python
+CONFIGURABLE_XBLOCKS_SETTINGS = {
+    "components": [{
+        "module": "lti_consumer",
+        "base_class": "ConfigurableLtiConsumerXBlock",
+        "subclasses": [{
+            "name": "Demo LTI Service",
+            "display": "Demo LTI xblock",
+            "lti": {
+                "key": "jisc.ac.uk",
+                "secret": "secret",
+            },
+            "default_values": {
+                "description": "Demo",
```

Comment:
@rmoch we discussed using another field. I don't think we should use this field. Did you test using the customer_parameters? or a new dedicated field that we add?